keylime/tenant.py

```diff
@@ -640,28 +643,29 @@ def do_cvlist(self):
 
         response = do_cvstatus.get(f"/v{self.api_version}/agents/?verifier={verifier_id}", timeout=self.request_timeout)
 
+        response_json = Tenant._jsonify_response(response, False, True)
```

Comment:
That is the 3rd call with reversed boolean parameters from the default.  Should be that a reversed defaults are better?

Reply:
I elected to keep the "newest" calls, on the allowlist API (near the end of the file), as unmodified as possible.